src/components/general/LaunchBanner/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react'
+
+import { Link } from 'react-router-dom'
+
+import Banner from '@/components/general/Banner'
+import ChevronRightIcon from '@/components/general/Icon/ChevronRightIcon'
+import { localStorageKeys } from '@/constants/localStorage'
+import { LocalStorage } from '@/utils/localStorage'
+
+import './LaunchBanner.scss'
+
+const LAUNCH_BANNER_URL = 'https://walletconnect.com/blog/introducing-web3inbox-app'
+const LAUNCH_BANNER_TERMS_URL = 'https://web3inbox.com/terms-soundwaves'
+
+const isBannerClosed = LocalStorage.get(localStorageKeys.launchBannerClosed) === 'true'
+
+export default function LaunchBanner() {
+  const [closed, setClosed] = useState(isBannerClosed)
+
+  if (closed) {
+    return null
+  }
```

Comment:
My preference for logic like this is having it be outside of where it's display  but I understand why putting it here makes it simpler. I'm okay with this because it's a temporary component